packages/tocco-util/src/form/formBuilder.js

```diff
@@ -135,19 +136,30 @@ export default (
   }
 
   const createLayoutComponent = (field, type, key, traverser) => {
-    if (type === layoutTypes.HORIZONTAL_BOX || type === layoutTypes.VERTICAL_BOX) {
-      const alignment = type === layoutTypes.HORIZONTAL_BOX ? 'horizontal' : 'vertical'
-      const children = traverser()
+    let elements = traverser()
 
-      if (children == null || (Array.isArray(children) && children.every(e => e == null))) {
-        return null
-      }
+    if (Array.isArray(elements)) {
+      elements = elements.filter(el => el)
+    }
 
-      return (
-        <LayoutBox key={key} label={field.label} alignment={alignment}>
-          {children}
-        </LayoutBox>
-      )
+    if (!elements || (Array.isArray(elements) && elements.every(e => e == null))) {
+      return null
+    }
+
+    const content = field.label
+      ? <Panel.Wrapper isFramed={true} isOpen={true}>
+        <Panel.Header><Typography.H4>{field.label}</Typography.H4></Panel.Header>
+        <Panel.Body>{elements}</Panel.Body>
+      </Panel.Wrapper>
+      : elements
+
+    if (type === layoutTypes.HORIZONTAL_BOX) {
```

Comment:
why not:

```
const layoutMap = {
   [layoutTypes.HORIZONTAL_BOX]: Layout.Container,
   [layoutTypes.VERTICAL_BOX]: Layout.Box
}
const LayoutComponent = layoutMap[type] 

if (LayoutComponent) {
  return <LayoutComponent key={key}>{content} </LayoutComponent >
} else {
      consoleLogger.logWarning(`Layout type "${type}" for box "${field.id}" is unknown.`)
      return <div key={key}>{content}</div>
 }	    

```
But this only if its a realistic scenario that "type" is neither a horizontal box nor a vertical box.
Otherwise:

```

const LayoutComponent = type === layoutTypes.HORIZONTAL_BOX ? Layout.Container : Layout.Box
 return <LayoutComponent key={key}>{content} </LayoutComponent >

```